ts/util/html-utils.ts

```diff
@@ -51,6 +52,7 @@ export function shadeElement(element: HTMLElement, status?: CellStatus | null) {
 
 export type TextStyle = "error";
 
+// styles cell text to match cell shading
```

Comment:
Not necessarily related to cells or cell shading. We can define different text styles here that may or may not go with any particular shading, and may also use the error style in other places without shading